<launch>
  <arg name="config_file" default="$(find pose_tracking)/config/dev_test_tracker.yaml"/>

  <!-- Pose Estimation Node-->
  <node name="dev_test_tracker" type="dev_test_tracker"
        pkg="pose_tracking" output="screen" respawn="false"> <!-- launch-prefix="valgrind" -->      
        <rosparam command="load" file="$(arg config_file)" />
        <param name="config_file"  type="string"  value="$(arg config_file)" />              
  </node>
  
  <node pkg="tf" type="static_transform_publisher" name="SIM_CAM_IR_base_link1" args="0 0 0 0 0 0  SIM_CAM /SIM_CAM_IR 100" />

</launch>
